updated FVM docs, added oFVM explanation (#20)
Velocimeter/docs: pages/oFVMmech.md

# Exploring Options with oFVM

> oFVM is a new options token that helps to reduce sell pressure on the native FVM token. It is earned by liquidity providers (LPs) who provide liquidity to the FVM-FTM pool. oFVM can be redeemed for FVM at a discount, or it can be locked up for a period of time in order to earn veFVM. veFVM is a governance token that allows holders to vote on the distribution of emissions.

Reviewer: ```suggestion
oFVM is a new options token that helps to reduce sell pressure on the native FVM token. It is earned by liquidity providers (LPs) who provide liquidity to the FVM-FTM pool. oFVM can be redeemed for FVM at a discount, or it can be locked up for a period of time as veFVM. veFVM is a governance token that allows holders to vote on the distribution of emissions, as well as receive weekly bribes and fees.
```

There are three main options for LPs who earn oFVM:

>  - FVM-FTM LP Lock Option: This option allows LPs to lock their oFVM rewards for a period of time. During this time, the oFVM rewards will be compounded back into the FVM-FTM pool, earning the LP additional FVM and FTM.

Reviewer: ```suggestion
 - FVM-FTM LP Lock Option: This option allows LPs to lock their oFVM rewards for a period of time. During this time, the oFVM rewards will be compounded back into the FVM-FTM pool, earning the LP additional oFVM and FTM.
```

>  - veFVM Option: This option allows LPs to redeem their oFVM rewards for veFVM. veFVM is a governance token that allows holders to vote on the distribution of emissions.

Reviewer: ```suggestion
 - veFVM Option: This option allows LPs to redeem their oFVM rewards for veFVM. veFVM is a governance token that allows holders to vote on the distribution of emissions and receive weekly bribes and fees.
```

## Benefits of oFVM
The introduction of oFVM has a number of benefits, including:
Reduced sell pressure: oFVM can be redeemed for FVM at a discount, which reduces the sell pressure on the native FVM token.
Increased liquidity: The FVM-FTM pool will be more liquid as a result of oFVM, as LPs will be incentivized to lock their oFVM rewards in the pool.
More sustainable APRs: The FVM-FTM pool will be able to offer more sustainable APRs as a result of oFVM, as the pool will be more liquid and there will be less sell pressure on the native FVM token.

## Conclusion
The introduction of oFVM is a positive development for the FVM ecosystem. oFVM has the potential to reduce sell pressure on the native FVM token, increase liquidity in the FVM-FTM pool, and offer more sustainable APRs for retail & protocols.

## Key terms
oFVM: A new token that is earned by LPs who provide liquidity to the FVM-FTM pool. oFVM can be redeemed for FVM at a discount, or it can be locked up for a period of time in order to earn veFVM.
FVM: The native token of the FVM ecosystem.
veFVM: A governance token that allows holders to vote on the distribution of emissions.

> LP: A liquidity provider. An LP is someone who provides liquidity to a liquidity pool.

Reviewer: need a list here instead of plain text. use minus signs for it
